views/routes.json

```diff
@@ -1,185 +1,213 @@
 {
```

Comment:
went to look at this file locally and realized that `*.json` is in the .gitignore. any chance we can add an exclusion for this file?

Reply:
also: just want to be sure that the changes in here are intentional? I'm assuming this is just from a difference in what your editor does for json. I don't mind one formatting over the other, but want to be sure we're not going to be swapping back and forth.

Reply:
I think `views/routes.json` is already excluded from the .gitignore. `*.json` excludes all JSON files at the root of the repo. `**.json` would exclude recursively. My editor (VS Code) is picking up the file without any issue.

For the formatting changes, I could roll them back but we should be removing this JSON file relatively soon, so I'm personally fine with a bit of thrashing.